lms/envs/test.py

```diff
@@ -656,7 +645,3 @@
 ########################## Derive Any Derived Settings  #######################
 
 derive_settings(__name__)
-
-FILE_UPLOAD_STORAGE_BUCKET_NAME = 'test_bucket'
-AWS_ACCESS_KEY_ID = 'test_access_Key'
-AWS_SECRET_ACCESS_KEY = 'test_secret_access_key'
```

Comment:
We might need these to run custom storage unit tests (philu's s3 tests only)

Reply:
Yes I've moved them at line `584`, to group all philu settings at a single point.